docs/src/environments/scalability.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Scale your live site
+sidebarTitle: Scalability
+description: Learn how to scale your live site.
+---
+
+Your production environment gets a [pool of resources](../create-apps/app-reference.md#sizes)
+based on your [plan size](../administration/pricing/_index.md).
+These resources are split up between the apps and services you've defined.
+
+You might occasionally get increased load on your site,
+such as during a major product launch or a sales event like Black Friday.
+This pressure might cause problems in your site or even bring it down.
+At such times, you want your site to have enough resources to handle the additional load.
+
+How your site handles the need to scale depends on your plan.
+On higher tiers with the Observability Suite,
+everything is taken care of you automatically.
```

Comment:
🛠️ ❓ The sentence is either missing the word _for_ before _you_, or the word _you_ is superfluous. Either way, I'm wondering if the sentence shouldn't be in the active voice instead, something like "Platform.sh takes care of everything on your behalf" or similar?

Reply:
I had the sentence active at the start, but it felt longer that way and `Platform.sh` didn't feel specific enough to really add a lot of help. I mean that it felt like saying `the system does X`, where `the system` isn't adding a lot of value. If you like it a lot more, it could be `Platform.sh takes care of everything for you`. (The mistaken `you` was left over from when I had the sentence like that before, I think.) I like active, but I also like short and specific, so not sure which works better. :shrug: